linux/Platform.c

```diff
@@ -1017,6 +1019,31 @@ bool Platform_init(void) {
    LibSensors_init();
 #endif
 
+   char target[PATH_MAX];
+   ssize_t ret = readlink(PROCDIR "/self/ns/pid", target, sizeof(target) - 1);
+   if (ret > 0) {
+      target[ret] = '\0';
+
+      if (!String_eq("pid:[4026531836]", target)) { // magic constant PROC_PID_INIT_INO from include/linux/proc_ns.h#L46
+         Running_containerized = true;
+         return true; // early return
+      }
+   }
+
+   FILE* fd = fopen(PROCDIR "/1/mounts", "r");
+   if (fd) {
+      char lineBuffer[256];
+      while (fgets(lineBuffer, sizeof(lineBuffer), fd)) {
+         // detect lxc or overlayfs and guess that this means we are running containerized
+         if (String_startsWith(lineBuffer, "lxcfs ") || String_startsWith(lineBuffer, "overlay ")) {
+            fprintf(stderr, "%s\n", lineBuffer);
+            Running_containerized = true;
+            break;
+         }
+      }
+      fclose(fd);
+   } // if (fd)
```

Comment:
No need to comment where this brace is coming from.
```suggestion
   }
```

Reply:
@BenBE Apparently I felt the need, stop wasting time please